source/mesh_deformation/fastscape.cc

```diff
@@ -301,8 +301,8 @@ namespace aspect
       if (dim == 2)
         {
           fastscape_dy = fastscape_dx;
-          fastscape_y_extent = std::round(fastscape_y_extent_2d/fastscape_dy)*fastscape_dy + fastscape_dy * ghost_nodes;
-          fastscape_ny = 1+fastscape_y_extent/fastscape_dy;
+          fastscape_y_extent = (std::round(fastscape_y_extent_2d/fastscape_dy) + ghost_nodes) * fastscape_dy;
+          fastscape_ny = 1+int(std::round(fastscape_y_extent_2d/fastscape_dy)) + ghost_nodes;
```

Comment:
Is ``int(x)`` a C style cast like ``(int)x``? If yes, maybe use ``static_cast``?

Reply:
It's like calling `std::vector<double>(17, 3.1415926)`: It creates an unnamed object of type `std::vector<double>`, with what is between the parentheses being the constructor arguments.

Either way, I'll change it to `static_cast<int>` later.